dashboard/src/scenes/auth/forgot.jsx

```diff
@@ -45,13 +50,25 @@ const View = () => {
     setResetForm((form) => ({ ...form, [e.target.name]: e.target.value }));
   };
 
+  const h1 = useMemo(() => {
+    if (newUser) return "Demander un nouveau lien de première connection";
+    if (newLinkRequest) return "Demander un nouveau lien de réinitialisation du mot de passe";
+    return "Réinitialiser le mot de passe";
+  }, [newLinkRequest, newUser]);
+
+  const description = useMemo(() => {
+    if (newUser) return "le lien de première connexion";
+    if (newLinkRequest) return "le nouveau lien de réinitialisation du mot de passe";
+    return "le lien de réinitialisation du mot de passe";
+  }, [newLinkRequest, newUser]);
+
   if (done) {
     return (
       <div className="tw-mx-10 tw-my-0 tw-w-full tw-max-w-lg tw-overflow-y-auto tw-overflow-x-hidden tw-rounded-lg tw-bg-white tw-px-7 tw-pb-2 tw-pt-10 tw-text-black tw-shadow-[0_0_20px_0_rgba(0,0,0,0.2)]">
-        <h1 className="tw-mb-6 tw-text-center tw-text-3xl tw-font-bold">Réinitialiser le mot de passe</h1>
+        <h1 className="tw-mb-6 tw-text-center tw-text-3xl tw-font-bold">{h1}</h1>
```

Comment:
ça reste une suggestion bien sûr ! J'évite de mettre du code d'affichage ailleurs. Là on peut dire "c'est des opérateurs ternaires pas lisibles" mais je trouve ça quand même plus simple, malgré ce problème.

```suggestion
        <h1 className="tw-mb-6 tw-text-center tw-text-3xl tw-font-bold">{
          newUser
            ? "le lien de première connexion"
            : newLinkRequest
            ? "le nouveau lien de réinitialisation du mot de passe"
            : "le lien de réinitialisation du mot de passe"
        }</h1>
```

Mais c'est une opinion, tu peux la jeter !